packages/wonder-blocks-link/components/link.js

```diff
@@ -19,7 +19,7 @@ export type SharedProps = {
      * Whether to display a caret after the text.
      */
     // TODO(yejia): Add once we have an Icon component
-    caret: boolean,
+    caret?: boolean,
```

Comment:
Probably better to have a `defaultProps` for this and have it be `false`.

Reply:
It's already in `defaultProps`... removing the `?`